Header header
int8 Service=0
int8 Observer=1
int8 serialMode